addon/components/ui-dropdown.js

```diff
@@ -1,17 +1,20 @@
 import Ember from 'ember';
 import layout from '../templates/components/ui-dropdown';
+import SettingsMixin from '../mixins/settings';
 
-export default Ember.Component.extend({
+export default Ember.Component.extend(SettingsMixin, {
   layout: layout,
   classNames: ['ui', 'dropdown'],
+  moduleName: 'dropdown',
   didInsertElement () {
     var component = this;
     var selected = this.get('selected');
-    this.$().dropdown({
+    this.setSettings({
```

Comment:
setSettings seems kind of confusing as a name.  How about something like `semanticBehaviours` ?